### PR TITLE
Added Arduino devices

### PR DESCRIPTION
**Pull request by karlsoderby**
#### Summary

...

#### Changes

- Added MKR WAN 1310
- Added Arduino Pro Gateway LoRa®
- Added Portenta Vision Shield LoRa®

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/arduino/index.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This example contains just one end device: windsensor. It is referenced here in the index.
```
Comment (johanstokking) [inline vendor/arduino/index.yaml:1]: You might want to remove this text here

**Review by johanstokking — APPROVED** [review]
Great, just one remark for next time, thanks @karlsoderby

**Review by johanstokking — CHANGES_REQUESTED** [review]
Oh wait... you're adding a gateway. That is not supported (yet).

Please remove the gateway, thanks!

**Comment by karlsoderby** [comment]
Hi Johan! Thanks for approving, I will take a look first thing in the
morning and submit a new PR without the gateway!

**Review by johanstokking — APPROVED** [review]

**Comment by johanstokking** [comment]
Thanks @karlsoderby !

**Pull request merged**